procedures/schedule.md

```diff
@@ -0,0 +1,21 @@
+# Council Schedule Tracking
+
+The Council maintains a document at <https://hackmd.io/@rust-leadership-council/H1bb7yPs2> to track upcoming and recurring events and deadlines.
```

Comment:
Is a hackmd the best for this? Is there no service we can use that at least gives some indication that a deadline has been reached?

Reply:
Over in https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Tracking.20deadlines.20and.20time-sensitive.20items there weren't any particular suggestions for something that would be better. The desired properties I was looking for:

* Be lightweight and easy for people to add entries.
* Not require a commercial service that we need to pay for, at least in the short term. Getting that spun up seems like more work than I'd like to deal with.
* Similar to the previous point, I'd prefer to minimize the amount of permissions management with new external services. 

I would like some kind of notification ability, though I'm not sure what that would look like or if it would be helpful. I don't really need or want a notification for everything. My main mitigation strategy here is to surface upcoming deadlines in the sync meeting. That should bring visibility in a lightweight fashion.

An alternative would be to somehow leverage GitHub, but I'm not sure if that would be better. Milestones aren't really geared for fine-grained dates, and don't have notifications. Projects also don't seem to support dates. We could build our own tooling, but I don't really want to. 

Even if we had a commercial service like Google calendar for example, I'm not sure if that is necessarily better. An entry in a calendar can come and go without any tracking if it was followed up on or resolved. With it being a document, we know if it has been addressed based on whether or not it is still on the list.

There are approximately one million task management programs out there, but I don't know of anything that is clearly worth adopting that doesn't bring more cost than benefit.